srcpkgs/deno/template

```diff
@@ -0,0 +1,17 @@
+# Template file for 'deno'
+pkgname=deno
+version=1.4.4
+revision=1
+build_style=cargo
+makedepends="python ccache"
```

Comment:
ccache is not appropriate for CI nor official builders. For local builds, configure xbps-src in etc/conf as described in README.md.

Reply:
Gonna change this.